iothub/device/tests/Authentication/ClientAuthenticationWithTokenRefreshTests.cs

```diff
@@ -240,6 +240,47 @@ public async Task ClientAuthenticationWithSakRefresh_WithModule_SharedAccessKeyC
             Assert.AreEqual(token1, token2.TokenValue);
         }
 
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey_Throws()
+        {
+            var builder = new SharedAccessSignatureBuilder()
+            {
+                Key = TestSharedAccessKey + "123",
+            };
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey1_Throws()
+        {
+            var TestSharedAccessKey_temp = TestSharedAccessKey.Replace('Y', '-');
```

Comment:
camelCase and no underscores for separating words.
```suggestion
            var testSharedAccessKeyTemp = TestSharedAccessKey.Replace('Y', '-');
```